assets/js/admin/course-general-sidebar.js

```diff
@@ -51,6 +51,7 @@ const CourseGeneralSidebar = () => {
 	const featured = meta._course_featured;
 	const prerequisite = meta._course_prerequisite;
 	const notification = meta.disable_notification;
+	const senfEnrollmentNotAllowed = meta._self_enrollment_not_allowed;
```

Comment:
Typo here in the variable name.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/7231/commits/5d4190999b77645e56fbeb185631b9da7cfa6cbc